src/index.d.ts

```diff
@@ -703,6 +902,35 @@ declare namespace admin.instanceId {
   }
 }
 
+/**
```

Comment:
This should be added to the `projectManagement()` method above to avoid repeating: https://firebase-dot-devsite.googleplex.com/docs/reference/admin/node/admin.projectManagement